lib/matobj1.gd

```diff
@@ -85,6 +84,58 @@
 DeclareCategory( "IsVectorObj", IsVector and IsCopyable );
 
 
+#############################################################################
+##
+##  <#GAPDoc Label="IsMatrixOrMatrixObj">
+##  <ManSection>
+##  <Filt Name="IsMatrixOrMatrixObj" Arg='obj' Type="category"/>
+##
+##  <Description>
+##  This filter is the <Q>common roof</Q> for <Ref Filt="IsMatrix"/> and
```

Comment:
I am not sure if "common roof" will be understood ? Perhaps we could borrow from the `IsListOrCollection` documentation, which starts like this:
> Several functions are defined for both lists and collections, [...]
> IsListOrCollection is a supercategory of IsList and IsCollection (that  is, all lists and collections lie in this category)